sys/kern/imgact_elf.c

```diff
@@ -1803,10 +1803,11 @@ core_write(struct coredump_params *p, const void *base, size_t len,
 }
 
 static int
-core_output(void *base, size_t len, off_t offset, struct coredump_params *p,
-    void *tmpbuf)
+core_output(void * __capability base, size_t len, off_t offset,
+    struct coredump_params *p, void *tmpbuf)
 {
 	int error;
+	void *base_vaddr = (void *)(uintptr_t)(uintcap_t)base;
```

Comment:
It's not though for a purecap kernel. Change the name of the function argument to `base_cap` or similar and call this one `base`? Will also reduce the diff.

Reply:
Will rename.